server/src/api/swagger.yaml

```diff
@@ -316,6 +316,27 @@ paths:
           schema:
             $ref: "#/definitions/RootCACert"
       x-swagger-router-controller: "Pki"
+  /environments/{envId}/status:
```

Comment:
this endpoint MUST receive the participant (DFSP) as a parameter, otherwise, we cannot know the status of which participant is requested

Reply:
Changed per review. Thank you